pyaerocom/colocateddata.py

```diff
@@ -81,55 +85,60 @@ class ColocatedData:
         if init fails
     """
 
-    __version__ = "0.11"
-
-    def __init__(self, data=None, **kwargs):
-        self._data = None
-        if data is not None:
-            if isinstance(data, Path):
-                # make sure path is str instance
-                data = str(data)
-            if isinstance(data, str):
-                self.open(data)
-            elif isinstance(data, xarray.DataArray):
-                self.data = data
-            elif isinstance(data, np.ndarray):
-                if not data.ndim in (3, 4):
-                    raise DataDimensionError("invalid input, need 3D or 4D numpy array")
-                elif not data.shape[0] == 2:
-                    raise DataDimensionError(
-                        "first dimension (data_source) must be of length 2(obs, model)"
-                    )
```

Comment:
Shouldn't these apply to DataArrays also?

Reply:
This is the old logic, which probably does need to be applied to xarray data objects since they can be make outside of pyaerocom now.